config/navigation.txt

```diff
@@ -43,7 +43,6 @@ Learn
 
     Runtime Environment
       [/runtime/runtime]
```

Comment:
yeah, its definitely not great. I have a plan to fix up all the URLs and restructure some of the menu, but it will be in a later PR, this one is more content focused for now